### PR TITLE
Update reviewer test apps for onboarding

### PR DESCRIPTION
**Pull request by eviljeff**

### CODE REVIEW

File: mkt/webapps/management/commands/generate_reviewer_apps.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from django.core.management.base import BaseCommand, CommandError
+
+from mkt.webapps.fakedata import generate_apps_from_specs
+
+
+class Command(BaseCommand):
+    """
+    Usage:
+
+        python manage.py generate_apps_from_spec <JSON spec file>
```
Comment (diox) [inline mkt/webapps/management/commands/generate_reviewer_apps.py]: Copy/paste error

**Comment by diox** [comment]
How is the new command used, why is it necessary ? I'm not familiar with reviewer onboarding stuff.

**Comment by eviljeff** [comment]
The command creates `n` sets of defined fake apps according to the specifications in the json file - each has known issues (or none) so we can have the new reviewers test them and review them as real apps (some are genuine apps, others test apps).  Someone with appropriate access for -dev would just need to run the command before we're ready to do the onboarding.  Then we can allocate #1's apps to Ali Babar, #2's apps to Claire Dude, #3's apps to Ed Fox, etc.

File: mkt/webapps/management/commands/generate_reviewer_apps.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+class Command(BaseCommand):
+    """
+    Usage:
+
+        python manage.py generate_reviewer_apps <JSON spec file>
+
+    """
+
+    help = 'Generate sets of prefixed reviewer apps for onboarding'
+    args = '<JSON spec filename> <number of sets>'
```
Comment (robhudson) [inline mkt/webapps/management/commands/generate_reviewer_apps.py]: Maybe update the docstring above which doesn't mention the sets.

File: mkt/webapps/management/commands/generate_reviewer_apps.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        sets = 1
+        if (len(args) > 1):
+            sets = int(args[1])
+
+        for x in xrange(sets):
+            generate_apps_from_specs(
+                specs,
+                os.path.abspath(os.path.dirname(args[0])),
+                0,
+                '#%s - ' % x)
```
Comment (robhudson) [inline mkt/webapps/management/commands/generate_reviewer_apps.py]: How does `sets` and `repeats` differ? I'm wondering why not use the `repeats` here?

File: data/apps/reviewer_apps.json [hunk 4]
```diff
@@ -1,26 +1,84 @@
...
+ },
+ {"type": "packaged",
+  "status": "pending",
+  "name": "CutTheRope",
+  "categories": ["games"],
+  "device_types": ["firefoxos", "desktop", "mobile"],
+  "privacy_policy": "http://www.cuttherope.net/privacy.html",
+  "preview_files": ["images/ctr1.png"],
+  "package_file": "manifests/ctr.zip",
+  "description": "Cut the rope to feed candy to little monster Om Nom®! 400 million downloads around the world of this phenomenal puzzle game. \n\nA mysterious package has arrived, and the little monster inside has only one request… CANDY! Collect gold stars, discover hidden prizes and unlock exciting new levels in this additively fun, award-winning, physics-based game!"
```
Comment (robhudson) [inline data/apps/reviewer_apps.json]: You'll need to update some of these categories now. E.g. 'books' no longer exists.

File: mkt/webapps/management/commands/generate_reviewer_apps.py (hunk 3 above)
Comment (eviljeff) [inline mkt/webapps/management/commands/generate_reviewer_apps.py]: I wanted to be able to prefix the name of each app in a copy (set) of the reviewer apps with a number but thought keeping the functionality of generate_apps_from_specs to same was easier (assuming anyone actually used it after QA asked for it).

Comment (eviljeff) [inline mkt/webapps/management/commands/generate_reviewer_apps.py]: I'm not wedded to the implementation here, fwiw.  It was just the easiest way I could think of making distinguishable apps for onboardees to pick out of the queue.

File: data/apps/reviewer_apps.json (hunk 4 above)
Comment (eviljeff) [inline data/apps/reviewer_apps.json]: done

**Comment by robhudson** [comment]
r+

**Pull request merged**